app/views/amends/AmendDirectExportPlasticPackagingView.scala.html

```diff
@@ -30,13 +30,12 @@
 @(form: Form[_], obligation: TaxReturnObligation)(implicit request: Request[_], messages: Messages)
 
 @hintContent = {
-    <p class="govuk-body">@messages("amendDirectExportPlasticPackaging.hint")</p>
+    <p class="govuk-body">@messages("amendDirectExportPlasticPackaging.para")</p>
```

Comment:
We will need to make the changes to these views like we did for returns, 
To bring the heading off of the form component  and use our custom one, which will allow you to have the p tag in the middle